cluster/gce/configure.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# CRI_CONTAINERD_HOME is the directory for cri-containerd.
+CRI_CONTAINERD_HOME="/home/cri-containerd"
+cd "${CRI_CONTAINERD_HOME}"
+
+# fetch_metadata fetches metadata from GCE metadata server.
+# Var set:
+# 1. Metadata key: key of the metadata.
+fetch_metadata() {
```

Comment:
Could we reuse the `fetch_metadata` and other duplicate codes of [test/configure.sh](https://github.com/kubernetes-incubator/cri-containerd/blob/master/test/configure.sh)?

Reply:
It's a bit hard. :(

`configure.sh` is passed to the node through GCE metadata. If we have multiple scripts for the initialization, we need to package most of them into the release tar, and refactor `configure.sh`.

I prefer doing that in the future. :) I'll file an issue for this.